users/autostart.rst

```diff
@@ -20,32 +20,141 @@ there are a number of easy solutions.
 Task Scheduler
 ~~~~~~~~~~~~~~
 
-#. Start the `Task Scheduler <https://en.wikipedia.org/wiki/Windows_Task_Scheduler>`__ (``taskschd.msc``).
-#. Create a New Task ("Action" menu -> "Create Task...").
-#. General Tab:
-    #. Name the task (for example 'Syncthing').
-    #. Check "Run whether user is logged on or not".
-    #. (recommended) Check "Do not store password. The task will only have access to local resources". Leave this option unchecked if you intend to store your folders on network drives and such.
-#. Triggers Tab:
-    #. Click "New...".
-    #. Set "Begin the task" to "At Startup".
-    #. (optional) Choose a delay.
-    #. Make sure Enabled is checked.
-    #. Click "OK".
-#. Actions Tab:
-    #. Click "New...".
-    #. [Action] should be set as "Start a program".
-    #. Enter the path to syncthing.exe in "Program/Script".
-    #. (optional) Enter the path to the parent folder of syncthing.exe in "Start in (optional)". This will allow you to use paths relative to this folder in Syncthing.
-    #. Click "OK".
+Task Scheduler is a built-in administrative tool, which can be used to
+start Syncthing automatically either at user log on, or at system
+startup. In both cases, Syncthing will open and stay invisible in
+background.
+
+You can launch Task Scheduler by going to ``Start Menu > Windows
+Administrative Tools`` and clicking on ``Task Scheduler``
+
+|Windows Task Scheduler Start Menu Screenshot|
+
+or by opening ``taskschd.msc`` from the Run command (``Win+R``).
+
+|Windows Task Scheduler Run Screenshot|
+
+For technical information about Task Scheduler visit
+https://docs.microsoft.com/windows/win32/taskschd
+
+Run at user log on with Task Scheduler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This option will make Task Scheduler automatically start Syncthing when you
+log on, and end it when you log out of your account.
+
+#. Create a new Task ("Actions" sidebar > "Create Task...").
+
+   |Windows Task Scheduler Screenshot|
+
+#. General tab:
+
+   #. Name the task (for example "Syncthing").
+
+   |Windows Task Scheduler General Logon Screenshot|
```

Comment:
The name field seems pretty unmistakable and in your face to me.

Reply:
Yes, that is true. I just wanted to add screenshots to each and every step for the sake of thoroughness. Should I remove this one?

Reply:
In my opinion yes. I don't see the value in the completeness alone, but I see value in having to scroll less. And here I think the added clarity is almost zero, at least below the annoyance of scrolling. Obviously that's personal bias as someone who gets on well with written instructions, so feel free to override me if you feel strongly about this.

Reply:
Actually, I would propose to keep these screenshots. The reason is that there is a  very important difference between

`|Windows Task Scheduler General Startup Screenshot|`

![image](https://user-images.githubusercontent.com/5626656/81473553-37ca8880-923a-11ea-96a7-f6b02eb1d311.png)

and

`|Windows Task Scheduler General Logon Screenshot|`

![image](https://user-images.githubusercontent.com/5626656/81473559-3f8a2d00-923a-11ea-99ea-c87308bf3a7c.png)

Of course, I am talking about the bottom part. I think that it is a good idea to have both of them, so that the difference between the two methods is clearly visible.

On the other hand, it would maybe be better to have small thumbnails instead of full screenshots, and then anyone could just click and view each of them in full when necessary. I do not think that we have this functionality in Sphinx though.

Reply:
Fine, all screenshots it is :)

There is https://pythonhosted.org/sphinxcontrib-images/ and that might be nice, but I don't expect you to do it - I have no clue how difficult it is and how well it would work.

Reply:
Yeah, I have also found some possible addons to Sphinx online, but I guess that they would require a server-side installationand configuration, right?

By the way, I have an idea in my head to merge the two Task Scheduler methods, so that most of the screenshots could be used only once, and for both. This way, we could reduce the number of duplicate screenshots, and also make the whole instructions shorter. I am going to try it in the next few days, and then update the commit.